search/django/weedid/tasks.py

```diff
@@ -35,12 +44,29 @@ def submit_upload_task(weedcoco_path, image_dir, upload_id):
             upload_id,
         )
     except Exception as e:
+        if not new_upload:
```

Comment:
I think the reader of the code would find it hard to understand why the data should be moved to the final dir in case of an exception.